### ISSUE TITLE
Unable to process RSS feed

### ISSUE THREAD

**Issue by LilDrunkenSmurf** [Qolors/FeedCord#5]
I setup an RSS reader pointing at 3 URLS, all with xml feeds, but it keeps spitting out errors for one of them:

https://www.leadedsolder.com/feed.xml
https://seatsafetyswitch.com/rss
https://carsmakeyoustupid.tumblr.com/rss

```
info: FeedCord.src.RssReader.FeedProcessor[0]
      Checking if any new posts for https://www.leadedsolder.com/feed.xml...
info: FeedCord.src.RssReader.FeedProcessor[0]
      Checking if any new posts for https://seatsafetyswitch.com/rss...
info: FeedCord.src.RssReader.FeedProcessor[0]
      Checking if any new posts for https://carsmakeyoustupid.tumblr.com/rss...
fail: FeedCord.src.Services.RssProcessorService[0]
      No items found in the RSS feed. FeedCord only supports Traditional RSS Feeds
warn: FeedCord.src.RssReader.FeedProcessor[0]
      Failed to fetch or process the RSS feed from https://www.leadedsolder.com/feed.xml.. Error Count: 1
info: FeedCord.src.Services.RssCheckerBackgroundService[0]
      Found no new posts. Ending background process..
info: FeedCord.src.Services.RssCheckerBackgroundService[0]
      Finished Background Process at 01/19/2024 12:36:16..
```

As far as I can tell, all the other RSS readers have no issues with the `feed.xml`, but for some reason feedcord doesn't consider it a traditional RSS feed.

**Comment by Qolors** [comment]
When making this I was writing my own parser for the RSS Feeds..

I'm working on integrating some tried and true libraries to give better parsing capabilities with your prior github request. 

So the URLs spitting out the error "no items found.." will most likely be fixed with that.

As far as the URLs throwing "failed to fetch or process" I will need to check these further to see where the issues lie.

Thanks for your patience!

**Comment by LilDrunkenSmurf** [comment]
Appreciate the quick response! Thanks so much!

**Comment by Qolors** [comment]
This has been tested & fixed with the latest [1.3.0 Release](https://github.com/Qolors/FeedCord/releases/tag/v1.3.0)

If there are any unhandled issues please reach out - thanks!
